gradio/helpers.py

```diff
@@ -263,6 +263,7 @@ async def load_example(example_id):
                 show_progress="hidden",
                 postprocess=False,
                 queue=False,
+                api_name="load_example",
```

Comment:
I added this in the process of fixing some tests. Makes testing more robust and can be useful for developers as well.